bpf/bpf_host.c

```diff
@@ -1384,7 +1386,7 @@ int cil_to_netdev(struct __ctx_buff *ctx)
 	}
 
 	switch (proto) {
-# if defined ENABLE_ARP_PASSTHROUGH || defined ENABLE_ARP_RESPONDER
+# if defined(ENABLE_IPV4) && (defined(ENABLE_ARP_PASSTHROUGH) || defined(ENABLE_ARP_RESPONDER))
```

Comment:
I think it's worth changing the agent code so that it doesn't define ENABLE_ARP_*  if IPv4 is disabled. This way we won't need this change and most of the rest below.